### PR TITLE
docs: mixins utilities PTL-1696

### PR DESCRIPTION
**Pull request by Gareth-Spencer-Genesis**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No

Have you done a trial build to check all new or changed links?
Yes

Is there anything else you would like us to know?
No

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```typescript
+import { LifecycleMixin } from '@genesislcap/foundation-utils';
+import {customElement, GenesisElement, observable} from '@genesislcap/web-core';
+
+@customElement({
+  name: 'my-element',
+  template: html`<!-- template here -->`,
+})
+export class MyExampleComponent extends LifecycleMixin(GenesisElement) {
+  // Your element's implementation
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx:44]: can we flesh this out with a simple example? Not even sure myself what we can do with this.

File: docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```typescript
+import { LifecycleMixin } from '@genesislcap/foundation-utils';
+import {customElement, GenesisElement, observable} from '@genesislcap/web-core';
+
+@customElement({
+  name: 'my-pending-element',
+  template: html`<!-- template here -->`,
+})
+export class MyExampleComponent extends PendingState(GenesisElement) {
+  // Element logic that involves asynchronous operations
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx:69]: same here

File: docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  - mixins
+  - augment
+  - addition
+  - lifecycle
+---
+
+The `mixins` category provides specialized functionalities that can be mixed into Genesis elements to enhance their capabilities and facilitate common patterns used within applications.
+
+:::info
+For Genesis code syntax only
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/014_utility-methods/08_mixins-util.mdx:18]:
```suggestion
Mixins are used for projects using Genesis syntax only
```

**Review by matteematt — APPROVED** [review]
Patrick is correct we really need to flesh out the examples, but I am happy to merge this for now and then we elaborate on them later. I can take up the example for LifecycleMixin but someone else can look at the other

**Pull request merged**